src/modules/common/AllLabelsContainer.tsx

```diff
@@ -39,12 +39,12 @@ const AllLabelsContainer = ({ labels, isSubmitted }: Props): JSX.Element => (
         isDraggingOver={dropSnapshot.isDraggingOver}
         id={ALL_LABELS_CONTAINER_ID}
       >
-        {labels.map((item, index) => (
-          <DraggableLabel
-            content={item.content}
-            draggableId={item?.id}
+        {labels.map((l, index) => (
+          <DraggableLabelToDroppableCont
```

Comment:
I've never paid much attention to the name, but I'm not sure it's a completely comprehensible name... Should we try and find something else?

Reply:
I don't understand either... It's not my component so I'll just let it there. Still need some refactor of the code.